root/report/ReleasesWithoutCaa.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * @flow strict
+ * Copyright (C) 2023 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import ReleaseList from './components/ReleaseList.js';
+import ReportLayout from './components/ReportLayout.js';
+import type {ReportDataT, ReportReleaseT} from './types.js';
+
+const ReleasesWithoutCaa = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportReleaseT>): React$Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report shows releases which have no cover art in the Cover Art
+       Archive at all. Most releases have some sort of cover art, so the
+       vast majority of these should eventually get some art uploaded to the
+       Cover Art Archive. We suggest restricting this report to your
+       subscriptions for a more manageable list of results.`,
```

Comment:
The second phrase here (_Most releases…_) is problematic as it encourages attaching cover art without caution. It happens too frequently that editors are wrongly attaching the cover art from a release (no matter whether it was or wasn’t in the MB database) to another release (in the MB database).

The third phrase is very relevant but should probably be even stronger as this report will always be huge.

Reply:
I extended the first bits, added a link to the how to docs (seemed useful) and split the last phrase onto a second help row, with stronger wording - does that seem better?